### ISSUE TITLE
Does anyone try to apply this project to BERT?

### ISSUE THREAD

**Issue by guotong1988** [tensorflow/model-optimization#29]
Thank you very much!

**Comment by raziel** [comment]
Not that we know of. I don't believe the open source bert trainer uses Keras. It would beb nice to see somebody trying it out though.

**Comment by alanchiao** [comment]
@haozha111 is trying this out

**Comment by Jasperty** [comment]
> @haozha111

how to apply this api to bert? do you train bert by keras? can we use bert trained by tensorflow?

**Comment by haozha111** [comment]
The pruning API only supports keras model right now. I'm just starting to trying it out, and don't have any code examples now. Will post here once I get some good results. Thanks!

**Comment by teijeong** [comment]
Hi @haozha111, is there some updates?

**Comment by lan2720** [comment]
> The pruning API only supports keras model right now. I'm just starting to trying it out, and don't have any code examples now. Will post here once I get some good results. Thanks!

Hi @haozha111, is there some updates?

**Comment by haozha111** [comment]
no, I haven't tried it recently. if you are interested, please follow docs in https://www.tensorflow.org/model_optimization to apply it to BERT, also cc'ing Jaesung who is currently working on MOT.
